tests/behat/features/cu_advanced_design_bundle/blocktitleicons.feature

```diff
@@ -0,0 +1,28 @@
+@AdvDesign
```

Comment:
I like `@AdvancedDesign` rather than abbreviated, but since that tag is used in a lot of other places, it's fine to leave as-is.

Reply:
Changed